harpy/diagnose.py

```
@@ -0,0 +1,44 @@
+def diagnose(directory):
+    """
+    Run the Snakemake debugger to identify hang-ups 
+    """
+    directory = directory.rstrip("/")
+    if not os.path.exists(f"{directory}/workflow/config.yaml"):
+        print_error("missing config file", f"Target directory [blue]{directory}[/blue] does not contain the file [bold]workflow/config.yaml[/bold]")
+        sys.exit(1)
+    with open(f"{directory}/workflow/config.yaml", 'r', encoding="utf-8") as f:
+        harpy_config = yaml.full_load(f)
+    
+    workflow = harpy_config["workflow"].replace(" ", "_")
```

Ruff (0.8.2)

Local variable `workflow` is assigned to but never used

Remove assignment to unused variable `workflow`

(F841)